tools/mo/openvino/tools/mo/back/ReverseInputChannels.py

```diff
@@ -556,8 +490,6 @@ def find_and_replace_pattern(self, graph: Graph):
         """
         Following transformations should run in strict order, that is why we disabled them all and run here 
         """
-        if graph.graph['cmd_params'].reverse_input_channels:
-            InsertReverseChannels().find_and_replace_pattern(graph)
         ReverseChannelsPropagationDown().find_and_replace_pattern(graph)
         ReverseChannelsPropagationUp().find_and_replace_pattern(graph)
```

Comment:
I see no code in MO now that will create `ReverseChannels` operation if you remove `InsertReverseChannels` transformation. It looks that these transformation become useless here.

Reply:
Here it is https://github.com/openvinotoolkit/openvino/blob/65ac9b5036e744359b7b07e7651904dc209ddae5/tools/mo/openvino/tools/mo/front/tf/UnpackPackReverseInputChannels.py#L67